virttest/test_setup.py

```diff
@@ -1543,8 +1565,8 @@ def sr_iov_cleanup(self):
 
         re_probe = False
         # if lsmod lists the driver then remove it to clean up
-        if not process.system('lsmod | grep %s' % self.driver,
-                              ignore_status=True, shell=True):
+        if not self.driver or not process.system('lsmod | grep %s' % self.driver,
```

Comment:
if `self.driver` is None, why do we remove it?

Reply:
You are right.  Updated, please help review again, thanks!